### ISSUE TITLE
Cannot load test data

### ISSUE THREAD

**Issue by nerdess** [CoderDojo/cp-local-development#42]
localhost:8000 is working fine but when I execute

> run ./localdev.js testdata zen

attached my postgres logfile 

[postgresql_error_log.txt](https://github.com/CoderDojo/cp-local-development/files/200833/postgresql_error_log.txt)

**Comment by nerdess** [comment]
I checked load_test_data.sh and found out the problem is within this line

`PGPASSWORD=$POSTGRES_PASSWORD psql --single-transaction -w -h $POSTGRES_HOST -U $POSTGRES_USERNAME -d $POSTGRES_NAME -f "$PROJECT_DIR/scripts/database/pg/populate-dojos.sql" --port $POSTGRES_PORT`

$POSTGRES_NAME uses the database "cp-dojos-development" but when I look into this database it has no tables so of course any INSERT state fails.

Where can I find the schema for cp-dojos-development?

**Comment by Wardormeur** [comment]
Migrations are ran/checked upon each run, try running ./localdev run zen once before using testdata

**Comment by nerdess** [comment]
there is no testdata, the database cp-dojos-development is emtpy...!

**Comment by Wardormeur** [comment]
The tables are created through migrations scripts which are run when the service is ran (ie : https://github.com/CoderDojo/cp-dojos-service/blob/master/service.js#L39 )
Even though you'll have the schema through the run, is disrecommend using test-data as https://github.com/CoderDojo/community-platform/issues/926 shows that a part of those are broken

**Comment by nerdess** [comment]
without test data there is no german version of the page so i cannot try and fix this issue: https://github.com/CoderDojo/community-platform/issues/856

also, without testdata i cannot login to try and replicate the bug. i can't even register as a new user on localhost since the "Select a user type" dropdown is not populated with the various coderdojo usertypes (Parent/Guardian, Mentor....)

i'd really love to fix some issues on the platform but without a working local dev environment i cannot do anything :(

**Comment by Wardormeur** [comment]
I'll try to have a look soon, but you don't need the test-data to run zen, running it with empty data is also possible
You'll need the translation file from crowdin to get it in german (https://crowdin.com/project/zen-community-platform/de#) and put it in the locale folder (under a newly created de_DE folder)

**Comment by nerdess** [comment]
but how can i login without the testdata? 

also: the translation file is several files/folders, not sure what is needed there....

**Comment by Wardormeur** [comment]
I don't believe you need any data to get it running,(as you can still register locally). Like I said, I'll have a look at it asap though.
In the meantime, can you tell me which version of postgres are you running?

**Comment by Wardormeur** [comment]
Hye, just tested here, testdata works when migrations are ran before, run ```./localdev run zen``` first, then the testdata one.
The reason is the run cmd is adding the tables which are required to insert the data in, and your error (which I also had by not runnning "run zen") specifically says you're missing the table initialization

**Comment by nerdess** [comment]
sorry for responding so late :)

did a git pull, ran npm install and ./localdev run zen and now the data migration worked...!

**Comment by nerdess** [comment]
well actually.....cheered to early. I was trying to login once using manager@example.com / test but then all just fell apart. when i execute ./localdev run zen i now get a strange error and localhost is not available:

Error: ENAMETOOLONG: name too long, stat 'workspace-zen/cp-e2e-tests/workspace-zen/cp-e2e-tests/workspace-zen/cp-e2e-tests/workspace-zen/cp-e2e-tests/workspace-zen/cp-e2e-tests/workspace-zen/cp-e2e-tests/workspace-zen/cp-e2e-tests/workspace-zen/cp-e2e-tests/workspace-zen/cp-e2e-tests/workspace-zen/cp-e2e-tests/workspace-zen/cp-e2e-tests/workspace-zen/cp-e2e-tests/workspace-zen/cp-e2e-tests/workspace-zen/cp-e2e-tests/workspace-zen/cp-e2e-tests/workspace-zen/cp-e2e-tests/workspace-zen/cp-e2e-tests/workspace-zen/cp-e2e-tests/workspace-zen/cp-e2e-tests/workspace-zen/cp-e2e-tests/workspace-zen/cp-e2e-tests/workspace-zen/cp-e2e-tests/workspace-zen/cp-e2e-tests/workspace-zen/cp-e2e-tests/workspace-zen/cp-e2e-tests/workspace-zen/cp-e2e-tests/workspace-zen/cp-e2e-tests/workspace-zen/cp-e2e-tests/workspace-zen/cp-e2e-tests/workspace-zen/cp-e2e-tests/workspace-zen/cp-e2e-tests/workspace-zen/cp-e2e-tests/workspace-zen/cp-e2e-tests/workspace-zen/cp-e2e-tests/workspace-zen/cp-e2e-tests/workspace-zen/cp-e2e-tests/workspace-zen/cp-e2e-tests/workspace-zen/cp-e2e-tests'
    at Error (native)

**Comment by nerdess** [comment]
did a "npm rebuild" and now get a different error which sums up as this: 

```
Seneca Fatal Error
==================

Message: Module version mismatch. Expected 48, got 47.
```

did "npm install --save seneca" but this does not help/change anything.....

**Comment by Wardormeur** [comment]
We didn't test on node 6.2 so far until we have proper tests, so that only 0.10.43 is the recognized version which runs properly
